### ISSUE TITLE
Search Engine Optimization

### ISSUE THREAD

**Issue by MaxHillebrand** [zkSNACKs/WasabiDoc#40]
As suggested in the [VuePress documentation](https://vuepress.vuejs.org/guide/markdown.html#front-matter), each markdown file can have some meta data in yaml format at the top of the file. This can include upon others a description and keywords. I'm not entirely sure how this will affect SEO only for the internal search box, or also for duck duck go searches.

As example the `/FAQ-UseWasabi.md`:

```
---
meta:
  - name: description
    content: Frequently Asked Questions how to use Wasabi Wallet.
  - name: keywords
    content: faq how-to use wasabi bitcoin wallet generate load receive send history coinjoin hardware settings coin-control 
---
```

**Comment by dennisreimann** [comment]
I'd suggest we use this for the "pillars" only. Otherwise this can easily be a lot of work for an unknown and potentially minor outcome. 
The search engines should be able to index the pages just fine as they are, so I wouldn't expect too much from such an optimization.

**Comment by MaxHillebrand** [comment]
We should do some testing if this is even useful at all, or if the default search engine does the job just fine.

**Comment by RiccardoMasutti** [comment]
SEO Specialist here. In my opinion is better to use JSON format.
Google’s John Mueller made it clear that Google (and most of other search engines as well) prefer JSON-LD structured data.

Source: https://www.searchenginejournal.com/google-structured-data-preference/297479/

**Comment by RiccardoMasutti** [comment]
In my opinion it is very important to optimize the microdata. We should first add them to the wasabiwallet.io site specifying, among others, the https://schema.org/SoftwareApplication schema

**Comment by MaxHillebrand** [comment]
closed by #139, more work needs to be done on the Algolia search engine #38.
